<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hadesvm::kernel::MountedFolderDialog</class>
 <widget class="QDialog" name="hadesvm::kernel::MountedFolderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="volumeNameLabel">
     <property name="text">
      <string>Volume name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="volumeNameLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="pathLabel">
     <property name="text">
      <string>Path:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="pathLineEdit"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="browsePushButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>8</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>hadesvm::kernel::MountedFolderDialog</receiver>
   <slot>_onOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>hadesvm::kernel::MountedFolderDialog</receiver>
   <slot>_onCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeNameLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>hadesvm::kernel::MountedFolderDialog</receiver>
   <slot>_onLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pathLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>hadesvm::kernel::MountedFolderDialog</receiver>
   <slot>_onLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browsePushButton</sender>
   <signal>clicked()</signal>
   <receiver>hadesvm::kernel::MountedFolderDialog</receiver>
   <slot>_onBrowsePushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_onOk()</slot>
  <slot>_onCancel()</slot>
  <slot>_onLineEditTextChanged(QString)</slot>
  <slot>_onBrowsePushButtonClicked()</slot>
 </slots>
</ui>
